src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
   // your code here
+  this.name = name;
+  this.age = age;
+  return this;
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  return object[property]? true : false;
 };
 
 const isOver65 = person => {
   // your code here
+  return person.age > 65? true : false;
 };
 
 const getAges = people => {
   // your code here
+  return people.map(ageArray => ageArray.age);
```

Comment:
Technically we are mapping over the people array here, so each iteration is using a person. I would change the variable name from `ageArray` to `person` to be more accurate.